crates/holochain/src/conductor/conductor.rs

```diff
@@ -12,12 +12,14 @@
 //! users in a testing environment.
 //!
 //! ```rust, no_run
-//! async fn async_main () {
+//!  async fn async_main () {
```

Comment:
is this space a typo?

Reply:
Yep, not sure how that got in there. I think the final closing brace not being included maybe confused the formatter? I don't remember touching this line otherwise. Anyway, fixed